[2022-07-31 00:05:39,507] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: trigger_rule_dag.task_3 scheduled__2022-07-30T00:00:00+00:00 [queued]>
[2022-07-31 00:05:39,521] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: trigger_rule_dag.task_3 scheduled__2022-07-30T00:00:00+00:00 [queued]>
[2022-07-31 00:05:39,521] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-31 00:05:39,522] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-07-31 00:05:39,522] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-31 00:05:39,543] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): task_3> on 2022-07-30 00:00:00+00:00
[2022-07-31 00:05:39,551] {standard_task_runner.py:52} INFO - Started process 17015 to run task
[2022-07-31 00:05:39,562] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'trigger_rule_dag', 'task_3', 'scheduled__2022-07-30T00:00:00+00:00', '--job-id', '268', '--raw', '--subdir', 'DAGS_FOLDER/trigger_rule_dag.py', '--cfg-path', '/tmp/tmpkmgvhz6b', '--error-file', '/tmp/tmp5rl36aub']
[2022-07-31 00:05:39,563] {standard_task_runner.py:80} INFO - Job 268: Subtask task_3
[2022-07-31 00:05:39,658] {task_command.py:369} INFO - Running <TaskInstance: trigger_rule_dag.task_3 scheduled__2022-07-30T00:00:00+00:00 [running]> on host 2525a2c6e2b8
[2022-07-31 00:05:39,753] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=trigger_rule_dag
AIRFLOW_CTX_TASK_ID=task_3
AIRFLOW_CTX_EXECUTION_DATE=2022-07-30T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-30T00:00:00+00:00
[2022-07-31 00:05:39,756] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-07-31 00:05:39,758] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'exit 0']
[2022-07-31 00:05:39,775] {subprocess.py:85} INFO - Output:
[2022-07-31 00:05:39,780] {subprocess.py:96} INFO - Command exited with return code 0
[2022-07-31 00:05:39,801] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=trigger_rule_dag, task_id=task_3, execution_date=20220730T000000, start_date=20220731T000539, end_date=20220731T000539
[2022-07-31 00:05:39,852] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-31 00:05:39,889] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
